index.js

```diff
@@ -37,11 +37,25 @@ server.route({
       reply(Boom.badRequest(validated.error.message));
     } else {
       var instance = validated.value;
-      reply(instance).code(201);
+      reply().code(201).header("Location: /v1/instances/5");
     }
   }
 });
 
+server.route({
+	method: 'GET'
+,	path: '/v1/instances/5'
```

Comment:
Basic ideas:

`/v1/instances/`: instances collection
    - GET - give me all instances
    - POST - create a new instance

`/v1/instances/{id}`: specific instance in the collection, with specified `id`
    - GET - give me a specific instance, with `id`
    - PUT - update an existing instance
    - DELETE - delete an instance

Not all of these methods are needed, but that's the general idea behind REST.